tardis/apps/publication_workflow/static/my-publications/my_publications.js

```diff
@@ -0,0 +1,133 @@
+     */
+    vm.loadDraftPubsData = function() {
+        draftPubsListRes.get().$promise.then(function (data) {
+                $log.debug("MyPublicationsController: Draft pubs list fetched successfully!");
+                vm.draftPubs = data;
+            },
+            function () {
+                $log.error("Could not load draft pubs list");
+            });
+    }
+
+    /**
+     * loadScheduledPubsData
+     *
+     * Fetch the list of scheduled pubs available to the user
+     */
+    vm.loadScheduledPubsData = function() {
+        scheduledPubsListRes.get().$promise.then(function (data) {
+                $log.debug("MyPublicationsController: Scheduled pubs list fetched successfully!");
+                vm.scheduledPubs = data;
+            },
+            function () {
+                $log.error("Could not load scheduled pubs list");
+            });
+    };
+
+    /**
+     * loadReleasedPubsData
+     *
+     * Fetch the list of released pubs available to the user
+     */
+    vm.loadReleasedPubsData = function() {
+        releasedPubsListRes.get().$promise.then(function (data) {
+                $log.debug("MyPublicationsController: Released pubs list fetched successfully!");
+                vm.releasedPubs = data;
+            },
+            function () {
+                $log.error("Could not load released pubs list");
+            });
+    };
+
+    /**
+     * loadRetractedPubsData
+     *
+     * Fetch the list of retracted pubs available to the user
+     */
+    vm.loadRetractedPubsData = function() {
+        retractedPubsListRes.get().$promise.then(function (data) {
+                $log.debug("MyPublicationsController: Retracted pubs list fetched successfully!");
+                vm.retractedPubs = data;
+            },
+            function () {
+                $log.error("Could not load retracted pubs list");
+            });
+    };
+
+    // Do initial data fetch
+    vm.loadDraftPubsData();
+    vm.loadScheduledPubsData();
+    vm.loadReleasedPubsData();
+    vm.loadRetractedPubsData();
+
+    vm.selected = [];
+    vm.limitOptions = [5, 10, 15];
+
+    vm.options = {
+      rowSelection: true,
+      multiSelect: true,
+      autoSelect: true,
+      decapitate: false,
+      largeEditDialog: false,
+      boundaryLinks: false,
+      limitSelect: true,
+      pageSelect: true
+    };
+
+    vm.query = {
+      order: '-id',
+      limit: 5,
+      page: 1
+    };
+
+    vm.toggleLimitOptions = function () {
+      vm.limitOptions = vm.limitOptions ? undefined : [5, 10, 15];
+    };
+
+    vm.loadStuff = function () {
+      vm.promise = $timeout(function () {
+        // loading
+      }, 2000);
+    }
+ 
+    vm.logItem = function (item) {
+      $log.debug(item.name + ' was selected');
+    };
+
+    vm.logOrder = function (order) {
+      $log.debug('order: ' + order);
+    };
+
+    vm.logPagination = function (page, limit) {
+      $log.debug('page: ' + page);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933951)
```suggestion
      $log.debug("page: " + page);
```